pages/Aliases and external links.md

```diff
@@ -0,0 +1,5 @@
+- It's also possible to link to pages outside of Logseq, i.e. websites. To point to a URL, either use the trigger menu by typing `/link` or use the keyboard shortcut `Cmd-l` (macOS)/`Ctrl-l` (Windows) to add the following shortcode to the block: `[]()`. Between the brackets (`[ ]`), put the label (name) of the link. Between the parenthesis, put the URL:
```

Comment:
Should also mention that when text or link url is selected, the shortcut will recognize it and move cursor input to the other part of the link.